src/app/core/services/label.service.ts

```diff
@@ -202,10 +202,11 @@ export class LabelService {
         return LabelService.severityLabels;
       case 'type':
         return LabelService.typeLabels;
-      case 'responseTag':
+      // case 'responseTag':
       case 'response':
         return LabelService.responseLabels;
     }
+    console.log('GetLabelList: Unfiltered Attr');
```

Comment:
For the logs, let's follow the format: ```code_file: action```
You can refer to https://github.com/CATcher-org/WATcher/issues/40

Reply:
I have updated the PR to use the LoggingService.